docs/source/deep-learning/pytorch/quickstart/pytorch_quickstart.ipynb

```diff
@@ -0,0 +1,1361 @@
+   "source": [
+    "print(f\"Image size: {training_data[0][0].shape}\")\n",
+    "print(f\"Size of training dataset: {len(training_data)}\")\n",
+    "print(f\"Size of test dataset: {len(test_data)}\")"
+   ],
+   "metadata": {
+    "colab": {
+     "base_uri": "https://localhost:8080/"
+    },
+    "id": "GlZkxQHu2t7V",
+    "outputId": "2ae9190a-03ac-4371-d6a9-0e9309a4e9de"
+   },
+   "execution_count": 4,
+   "outputs": [
+    {
+     "output_type": "stream",
+     "name": "stdout",
+     "text": [
+      "Image size: torch.Size([1, 28, 28])\n",
+      "Size of training dataset: 60000\n",
+      "Size of test dataset: 10000\n"
+     ]
+    }
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "We wrap the dataset into `Dataloader` for batching purposes. `Dataloader` is a useful tool for data preprocessing, for more details, you can refer to the [developer guide from PyTorch](https://pytorch.org/tutorials/beginner/basics/data_tutorial.html#preparing-your-data-for-training-with-dataloaders)."
+   ],
+   "metadata": {
+    "id": "NhV1htwx3qJR"
+   }
+  },
+  {
+   "cell_type": "code",
+   "source": [
+    "train_dataloader = DataLoader(training_data, batch_size=64)\n",
+    "test_dataloader = DataLoader(test_data, batch_size=64)"
+   ],
+   "metadata": {
+    "id": "RoZj28jT3N_n"
+   },
+   "execution_count": 5,
+   "outputs": []
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "## Define our Model\n",
+    "\n",
+    "Now let's define our model, we will build a simple convolutional neural network as the classifier. To define a PyTorch model, you will need to subclass from `torch.nn.Module` and override `__init__` to define model components, and `forward()` method to implementation the forward-pass logic.\n",
+    "\n",
+    "We use two convolutional layer and the model output will be the logits of each class (10 classes in total). Applying softmax on logits yields the probability distribution across classes."
+   ],
+   "metadata": {
+    "id": "J4XB9_TS4Gkc"
+   }
+  },
+  {
+   "cell_type": "code",
+   "source": [
+    "class ImageClassifier(nn.Module):\n",
+    "    def __init__(self):\n",
+    "        super().__init__()\n",
+    "        self.model = nn.Sequential(\n",
+    "            nn.Conv2d(1, 8, kernel_size=3),\n",
+    "            nn.ReLU(),\n",
+    "            nn.Conv2d(8, 16, kernel_size=3),\n",
+    "            nn.ReLU(),\n",
+    "            nn.Flatten(),\n",
+    "            nn.LazyLinear(10),  # 10 classes in total.\n",
+    "        )\n",
+    "\n",
+    "    def forward(self, x):\n",
+    "        return self.model(x)"
+   ],
+   "metadata": {
+    "id": "5FMb4gMK4ECL"
+   },
+   "execution_count": 6,
+   "outputs": []
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "## Connect to MLflow Tracking Server\n",
+    "\n",
+    "Before implementing the training loop, we need to configure the MLflow tracking server because we will log data into MLflow during training.\n",
```

Comment:
We can replace this entire block with the page reference here: https://www.mlflow.org/docs/latest/getting-started/running-notebooks/index.html

Reply:
Personally I really like an integrated experience while reading through guides/tutorials, which means the guide/tutorial is the only doc I need to read to get the work done. If it is a ipython notebook, I should be able to click "run all" and wait for the result (filling some prompted info is required tho, like credentials or so).